### PR TITLE
Moving from json to yaml configuration. Small changes in FO scraper

### LINKED ISSUE #30: Move to yaml
In the future, YAPO will make use of YAML for other purposes. For this reason, we want to move the current configuration file from JSON to YAML.

- A function to convert the old configuration file to YAML
- A function to write configuration values by name and value
- A function to read configuration values by name

### PR DESCRIPTION
**Pull request by cooperdk**
Function to store and retrieve settings from config/settings.yml (see videos/aux_functions)

### CODE REVIEW

File: videos/scrapers/freeones.py [hunk 1]
```diff
@@ -531,6 +531,7 @@ def search_freeones(actor_to_search, alias, force):
                             piercings = str(piercings).strip()
                             piercings = piercings.replace(";", ",")
                             piercings = piercings.replace(" and ", ",")
+                            if any([piercings.lower() == "none", piercings.lower() == "no piercings", piercings.lower() == "no"]): piercings="No piercings"
```
Comment (Merodac) [inline videos/scrapers/freeones.py:534]: never place the execution block of an if in the same line as the if.

File: videos/aux_functions.py [hunk 2]
```diff
@@ -1,10 +1,40 @@
...
 import urllib.request
 from videos.models import Actor, Scene, ActorTag
-import videos.views
+#import videos.views
+import YAPO.settings as ysettings
+import videos.const as constx
 
-def progress(count, total, suffix=''):
+
+def saveconf (yvar, yval):
```
Comment (Merodac) [inline videos/aux_functions.py:13]: use type hints. i'll write that only once, but of course it should be applied everywhere

File: videos/aux_functions.py [hunk 3]
```diff
@@ -1,10 +1,40 @@
...
+    yaml_dict = {}
+    configfile = ysettings.CONFIG_YML
+
+    if not path.isfile(configfile):
+        pathlib.Path(configfile).touch()
+
+    with open(configfile) as f:
+        yaml_dict = yaml.safe_load(f)
+        if yaml_dict is None:
```
Comment (Merodac) [inline videos/aux_functions.py:22]: use
`yaml_dict = yaml.safe_load(f) or {}`
if you want to ensure that yaml_dict is not None

File: videos/aux_functions.py [hunk 4]
```diff
@@ -1,10 +1,40 @@
...
+import YAPO.settings as ysettings
+import videos.const as constx
 
-def progress(count, total, suffix=''):
+
+def saveconf (yvar, yval):
+    yaml_dict = {}
+    configfile = ysettings.CONFIG_YML
+
+    if not path.isfile(configfile):
```
Comment (Merodac) [inline videos/aux_functions.py:17]: invert the if, open the file for reading only if it does not exist, but open the file for reading always -> no need for touch

File: videos/aux_functions.py [hunk 5]
```diff
@@ -1,10 +1,40 @@
...
+
+    yaml_dict[yvar] = yval
+    with open(configfile, 'w') as f:
+        yaml_dict = yaml.dump(yaml_dict, stream=f, default_flow_style=False)
+
+
+def readconf (configfile, variable):
+    with open(configfile, 'r') as file:
+        yaml_dict = yaml.safe_load(file)
+    value = yaml_dict[variable]
```
Comment (Merodac) [inline videos/aux_functions.py:33]: if variable is not found --> book, exception

File: videos/const.py [hunk 6]
```diff
@@ -3,7 +3,7 @@
 VIDEO_ROOT = "videos/"
 SHEET_ROOT = "videos/videosheet"
 TEMP_PATH = "videos/ffmpeg/temp/"
-VLC_PATH = ""
+VLC_PATH = "c:\\program files\\videolan\\vlc\\vlc.exe"
```
Comment (Merodac) [inline videos/const.py:6]: that is definitely no constant. that is a default value at most, but most propably a configuration settings anyway.

File: videos/scrapers/freeones.py (hunk 1 above)
Comment (cooperdk) [reply videos/scrapers/freeones.py:534]:
```suggestion
                            if any([piercings.lower() == "none", piercings.lower() == "no piercings", piercings.lower() == "no"]):
                                piercings="No piercings"
```

File: videos/aux_functions.py (hunk 2 above)
Comment (cooperdk) [reply videos/aux_functions.py:13]:
```suggestion
def saveconf (yvar: str, yval: str):
```

File: videos/const.py (hunk 6 above)
Comment (cooperdk) [reply videos/const.py:6]: I agree, but it was how it was originally written. There is a better way to do this.

File: videos/startup.py [hunk 7]
```diff
@@ -237,18 +229,48 @@ def recursive_add_folders(parent, folders, scene_to_add, path_with_ids):
...
+            auxf.saveconf("last_all_scene_tag", str(content["last_all_scene_tag"]))
+            print(f"\n███ Your JSON configuration was converted to YAML: ({src})! ███")
+
+        else:
+            auxf.saveconf("settings_version", vc.SETTINGS_VERSION)
+            auxf.saveconf("vlc_path", vc.VLC_PATH)
+            auxf.saveconf("last_all_scene_tag", vc.LAST_ALL_SCENE_TAG)
+            print(f"\n███ A new YAML configuration file was written to ({src})! ███")
+
+def getStarted ():
```
Comment (Merodac) [inline videos/startup.py:265]: in python, methods and classes should not be camel-cased, the words should be separated by an underscore

File: videos/startup.py [hunk 8]
```diff
@@ -237,18 +229,48 @@ def recursive_add_folders(parent, folders, scene_to_add, path_with_ids):
...
-    #    add_scene_to_folder_view(scene)
-    maybeMoveConfigJson()
+        if path.isfile(settings.CONFIG_JSON):
+            jsonconf = True
+            jsonsrc = settings.CONFIG_JSON
+
+        if jsonconf == True:
+
+            with open(jsonsrc, 'r') as fx:
+                xx = fx.read()
```
Comment (Merodac) [inline videos/startup.py:251]: it is only used once - but 'xx' is really a bad name for a variable

**Pull request merged**